### ISSUE TITLE
Live reloading app on each changes for developing mode

### ISSUE THREAD

**Issue by samialmorshedi** [angular/universal-starter#607]
I have read whole the readme file but there are just 3 deployment ways. One for client-side rendering, 2 others are for production mod. But I need another way for run application client-side and server-side together in live mode for developing. Please help me is there any solution for it.
How can I develop my application by universal on both sides?

**Comment by chintan13** [comment]
I was having same issue I am using three terminal with below commands

ng build --target=development --environment=prod --aot --watch
ng build --target=development --environment=prod --aot --watch --app 1
npm run webpack:server && npm run serve:ssr    // make stop/start this one only

so first two commands will be in watch mode and will update build automatically after changes.
so once builds are done you need to stop/start third command "npm run webpack:server && npm run serve:ssr "

with this at least its little faster in development

**Comment by chintan13** [comment]
yes it works for my setup, it seems your setup is little different,

here idea is to run build command in watch mode and then stop/start webpack command.
you should modify your commands accordingly

**Comment by chintan13** [comment]
things been changed much in this demo, I can see your problem now, sorry it will take little time for me to figure it out with this code.

**Comment by Gorniv** [comment]
look https://github.com/Angular-RU/angular-universal-starter/blob/master/package.json#L14
